src/layouts/webapp/Messages/ChatDetail.tsx

```
@@ -0,0 +1,160 @@
+                    className="size-[120px] object-contain"
+                    src={each.stickerUrl ?? ''}
+                  />
+                </div>
+              );
+            }
+            return (
+              <MessageItem key={each.id} type={each.direction}>
+                {each.msg}
+              </MessageItem>
+            );
+          })}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)